WISE-Community/WISE
# Updated maven dependencies and removed unused dependencies

Also fixed issue where lombok getters and setters were throwing compile-time errors in vscode.  

To test:
- make sure that you do not see any compile-time errors in VSCode
- run mvn test and make sure that all tests pass.
- make sure that WISE still works as before.

Closes #1959
